padglobal/padglobal.py

```diff
@@ -277,7 +278,7 @@ def list_or_none(l):
         await padinfo_cog.refresh_index()
         await ctx.send('finished reload')
 
-    @commands.group(aliases=['pdg'])
+    @commands.group(aliases=['pdg','pg'])
```

Comment:
Don't add top-level aliases unless they're super unique, definitely not short ones like this. I suggest removing both of these. You can always create a server-specific alias for padglobal.

There's a limited namespace for top level stuff; we just bumped into a problem where we can't use ^scheduler because the Scheduler cog also includes ^remindme.

Reply:
done